src/cdn_definitions/_impl/__init__.py

```diff
@@ -1,23 +1,29 @@
 import json
 import os
-
-
-def load_data():
-    # Load data from first existing of these
-    candidate_paths = [
-        os.path.join(os.path.dirname(os.path.dirname(__file__)), "data.json"),
-        "/usr/share/cdn-definitions/data.json",
-    ]
-
-    # If env var is set, it takes highest precedence
-    if "CDN_DEFINITIONS_PATH" in os.environ:
-        candidate_paths.insert(0, os.environ["CDN_DEFINITIONS_PATH"])
-
-    existing_paths = [p for p in candidate_paths if os.path.exists(p)]
-    return json.load(open(existing_paths[0]))
-
-
-DATA = load_data()
+import requests
+
+
+def load_data(source=None):
+    if source is None:
+        # Load data from first existing of these
+        candidate_paths = [
+            os.path.join(os.path.dirname(os.path.dirname(__file__)), "data.json"),
+            "/usr/share/cdn-definitions/data.json",
+        ]
+
+        # If env var is set, it takes highest precedence
+        if "CDN_DEFINITIONS_PATH" in os.environ:
+            cdn_definitions_path = os.environ["CDN_DEFINITIONS_PATH"]
+            candidate_paths.insert(0, cdn_definitions_path)
+            if cdn_definitions_path.startswith("http"):
+                return requests.get(cdn_definitions_path)
+        existing_paths = [p for p in candidate_paths if os.path.exists(p)]
+        return json.load(open(existing_paths[0]))
```

Comment:
I'm probably using an inefficient git workflow, but because I frequently squash and rebase my changes, I find that it's much easier and less confusing to work with a separate branch per commit, rather than with several commits on one branch. So, I would prefer to handle this small change in a separate PR, and rebase this PR's commits on top of that change after it's merged: Ihttps://github.com/release-engineering/cdn-definitions/pull/21